Example/Sources/Playgrounds/Avatar.playground/Contents.swift

```diff
@@ -6,7 +6,11 @@ import PlaygroundSupport
 //Get an image
 let testImage = #imageLiteral(resourceName: "NiceSelfi.jpg")
 
-var avatarView = AvatarView()
+var avatarView = AvatarView(frame: CGRect(x: 0, y: 0, width: 30, height: 30))
+avatarView.set(font: UIFont(name: "SinhalaSangamMN-Bold", size: 32)!)
+avatarView.minimumScaleFactor = 0.5
```

Comment:
but things like this `avatarView.minimumScaleFactor = 0.5` should be the default with the ability to be overwritten. IE it should not take 6 lines to setup the avatar.